apps/filecopy/src/site/conf/cluster-memory-conf.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
These are not new parameters. Existing properties in the operators are exposed with getters, setters.